src/main/java/uk/gov/hmcts/reform/sscs/util/SscsOcrDataUtil.java

```diff
@@ -0,0 +1,78 @@
+package uk.gov.hmcts.reform.sscs.util;
+
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+import java.time.format.DateTimeParseException;
+import java.util.List;
+import java.util.Map;
+
+public class SscsOcrDataUtil {
+
+    private static final String YES = "Yes";
+    private static final String NO = "No";
+
+    private SscsOcrDataUtil() {
+
+    }
+
+    public static Boolean hasPerson(Map<String, Object> pairs, String person) {
+
+        return findBooleanExists(getField(pairs,person + "_title"), getField(pairs,person + "_first_name"),
+            getField(pairs,person + "_last_name"), getField(pairs,person + "_address_line1"), getField(pairs,person + "_address_line2"),
+            getField(pairs,person + "_address_line3"), getField(pairs,person + "_address_line4"), getField(pairs,person + "_postcode"),
+            getField(pairs,person + "_date_of_birth"), getField(pairs,person + "_ni_number"),  getField(pairs,person + "_company"));
+    }
+
+    public static boolean findBooleanExists(String... values) {
+        for (String v : values) {
+            if (v != null) {
+                return true;
+            }
+        }
+        return false;
+    }
+
+    public static String getField(Map<String, Object> pairs, String field) {
+        return pairs.get(field) != null ? pairs.get(field).toString() : null;
+    }
+
+    public static boolean doBooleansContradict(Map<String, Object> pairs, List<String> errors, String value1, String value2) {
+        if (pairs.containsKey(value1) && pairs.containsKey(value2)) {
+            if ((boolean) pairs.get(value1) == (boolean) pairs.get(value2)) {
```

Comment:
haven't tested it, but is it possible to secure type conversion by following snippet?
```java
if (pairs.get(value1).equal(pairs.get(value2)) && pairs.get(value1) instanceof Boolean) {
    ...
}
```

just trying to avoid unnecessary type cast errors here

Reply:
Done